content/applications/studio/automated_actions.rst

```diff
@@ -1,249 +1,297 @@
 
-Define on which records of the model the automated action should be applied. It works the same way
-as when you apply filters on a model.
+.. note::
+   The :guilabel:`On UI change` trigger can only be used with the
+   :ref:`studio/automated-actions/action/python-code` action and only works when a modification is
+   made manually. The action is not executed if the field is changed through another automation
+   rule.
 
-.. _studio/automated-actions/action:
+.. _studio/automated-actions/trigger/external:
 
-Action
-======
+External
+--------
 
-Determine what the automated action should do (server action). There are eight types of action to
-choose from.
+Trigger automated actions based on an external event using a webhook. A webhook is a method of
+communication between two systems where the source system sends an HTTP(S) request to a destination
+system based on a specific event. It usually includes a data payload containing information about
+the event that occurred.
 
-.. _studio/automated-actions/action/python-code:
+To configure the :guilabel:`Webhook` trigger, enter the :guilabel:`URL` of the source system
+(i.e., the system sending the request), then enter the code to run to determine the record on which
+the automation rule should be executed in the :guilabel:`Target Record` field.
 
-Execute Python Code
--------------------
+.. note::
+   The code defined by default in the :guilabel:`Target Record` field works for webhooks coming from
+   another Odoo database.
 
-The action is used to execute Python code. The available variables are described on the
-:guilabel:`Python Code` tab, which is also used to write your code, or on the :guilabel:`Help` tab.
+You can also choose to :guilabel:`Log Calls` to record the payloads received, e.g., to make sure the
+data sent by the source system matches the expected format and content. This also helps identify
+and diagnose any issues that may arise. To access the logs, click the :guilabel:`Logs` smart
+button at the top of the :guilabel:`Automation rules` form.
 
-- To allow the action to be run through the website, tick :guilabel:`Available on the Website` and
-  add a :guilabel:`Website Path`.
+.. note::
+   If you wish to use the webhook's content for a purpose other than to find the record, you can
+   only do so using an :ref:`studio/automated-actions/action/python-code` action.
 
-.. _studio/automated-actions/action/new-record:
+.. _studio/automated-actions/action:
 
-Create a new Record
--------------------
+Actions
+=======
 
-The action is used to create a new record on any model.
+Once you have defined the automation rule's :ref:`trigger <studio/automated-actions/trigger>`, click
+:guilabel:`Add an action` to define the action to be executed.
 
-.. note::
-   Selecting a :guilabel:`Target Model` is only required if you want to target another model than
-   the one you are on.
+.. tip::
+   You can define multiple actions for the same trigger/automation rule.
```

Comment:
they are run in order (e.g. if you have an action to update a record then send an email, the email will use the updates values; if you set the email action before the update action, the email will use the values before the update action is run)